app/admin/inbox/test/page.tsx

```diff
@@ -43,49 +41,34 @@ export default function TestLambdaPage() {
 
   return (
     <div className="max-w-2xl w-full">
-      <h2 className="text-xl font-bold mb-4">Test Lambda Email Processing</h2>
-      <p className="text-sm text-gray-600 mb-6">
-        Test the Lambda function with a preview branch by sending a simulated email.
-      </p>
+      <h2 className="text-xl font-bold mb-4">Test Email Processing</h2>
       
       <form onSubmit={handleSubmit} className="flex flex-col gap-4">
-        <div>
-          <label htmlFor="previewBranchName" className="block mb-1">
-            Preview Branch Name *
-          </label>
-          <input
-            type="text"
-            id="previewBranchName"
-            name="previewBranchName"
-            required
-            placeholder="feature-branch-name"
-            className="w-full p-2 border rounded text-black"
-          />
-        </div>
-        
         <div>
           <label htmlFor="testSubject" className="block mb-1">
-            Test Subject (optional)
+            Test Subject *
           </label>
           <input
             type="text"
             id="testSubject"
             name="testSubject"
-            placeholder="Will auto-generate with [PREVIEW: branch-name] if empty"
+            required
+            placeholder="Enter test email subject"
             className="w-full p-2 border rounded text-black"
           />
         </div>
         
         <div>
           <label htmlFor="testSender" className="block mb-1">
-            Test Sender Email (optional)
+            Test Sender Email *
           </label>
-          <input
-            type="email"
+          <textarea
```

Comment:
Sorry, this should be just an input, it's the email body that we want to be a textarea